src/sentry/static/sentry/app/icons/iconClose.tsx

```diff
@@ -12,7 +12,7 @@ export const IconClose = React.forwardRef(function IconClose(
   }: IconProps,
   ref: React.Ref<SVGSVGElement>
 ) {
-  const color = providedColor;
+  const color = theme[providedColor] ?? providedColor;
```

Comment:
cc @doralchan - should all of our icons support this?

Reply:
@billyvg yes I checked with her :) I will open a PR updating all the icons 😉